docs/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -36,18 +36,19 @@ For each RC published in each stage:
 
 - version string in `version.go` has been updated (in the `release-vX.Y.Z` branch).
 - new commits should be added to the `release-vX.Y.Z` branch from `master` using `git cherry-pick -x ...`
+- `release-` branches are protected. You can do all needed updates on a separated branch and when everything is settled push to `release-vX.Y.Z`
```

Comment:
should this be a subitem of the above bullet-point?

Reply:
```suggestion
  - `release-` branches are protected. You can do all needed updates on a separated branch and when everything is settled push to `release-vX.Y.Z`
```